docs/sources/setup/install/docker.md

```diff
@@ -9,55 +9,120 @@ weight: 400
 # Install Loki with Docker or Docker Compose
 
 You can install Loki and Promtail with Docker or Docker Compose if you are evaluating, testing, or developing Loki.
-For production, Grafana recommends installing with Tanka or Helm.
+For production, Grafana recommends installing with Helm or Tanka.
 
-The configuration acquired with these installation instructions run Loki as a single binary.
+The configuration files associated with these installation instructions run Loki as a single binary.
 
 ## Prerequisites
 
 - [Docker](https://docs.docker.com/install)
 - [Docker Compose](https://docs.docker.com/compose/install) (optional, only needed for the Docker Compose install method)
 
-## Install with Docker
+## Install with Docker on Linux
```

Comment:
What's specific about Linux here? Will these instructions not work on Mac/BSD/etc?

Reply:
Good question, I was trying to differentiate between this and the Windows installation.  Any suggestions for a better heading?

Reply:
Actually, that was already the heading....

Reply:
The heading was previously _Install with Docker_ not _Install with Docker on Linux_ unless I'm missing something?

Reply:
![InstallDocker](https://github.com/grafana/loki/assets/4106682/dd9c6b60-432c-4df7-9153-6eebcb2ed005)
Previous heading arrangement ^